extensions/amp-facebook/0.1/amp-facebook.js

```diff
@@ -51,8 +53,15 @@ class AmpFacebook extends AMP.BaseElement {
   layoutCallback() {
     const iframe = getIframe(this.win, this.element, 'facebook');
     this.applyFillContent(iframe);
+    const sentinel = generateSentinel(window);
+    iframe.name = encodeURI(JSON.stringify({
```

Comment:
sorry, I'm confused at what needs to be done to fix this. The name attribute is totally random, because it contains the sentinel (which has a random string as part of it).